src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactComponent as CheckSvg } from '../../assets/icons/Check.svg';
+
+import { CheckboxProps } from './types';
+import { Div, Label, LabelSquare } from './styles';
+
+
+const CheckBox = ({
+                    gap,
+                    text,
+                    margin,
+                    onClick,
+                    fontSize,
+                    checked = false,
+                  }: CheckboxProps) => (
```

Comment:
We need to configure the formatting
If you use webstorm, it has its own formatting